### ISSUE TITLE
WIP: Dont group `Objects.yaml` by package name

### ISSUE THREAD

**Issue by mhsdesign** [neos/flow-development-collection#3321]
Was that always the case? I mean why the hell do we separate the object configuration by package LOL. But that's for another PR / ticket

_Originally posted by @kitsunet in https://github.com/neos/flow-development-collection/pull/3318#discussion_r1489399440_

**Comment by kitsunet** [comment]
I think I know why, because we use that information in proxy building, BUT a) I am not even sure it needs to be that way (apart from debugging information, but I have some ideas regarding that) and b) We could still provide it differently at runtime. Sounds weird but that could actually make sense, aaaaand STOP all that, writing this down actually gave me the answer.....

For the respective use case we already HAVE a way better way to do this, namely:

`\Neos\Flow\ObjectManagement\ObjectManagerInterface::getClassNameByObjectName()`, this should just return the _right_ implementation as in, the chosen one the objectmanager would inject if you asked it, and that's all we need and we have an object manager, at least compiletime which also can answer this question.

**Comment by mhsdesign** [comment]
Yes thanks wo i  was just too blind and sure there is no better way 😅 Implemented here: https://github.com/neos/flow-development-collection/pull/3322

But this issue may persist for that we had many WTFs why the config is not merged and this will possibly make the array bigger?

**Comment by mhsdesign** [comment]
This is a duplicate of the better well formed issue by karsten :D https://github.com/neos/flow-development-collection/issues/3237
